tests/validate/verify_test/src/host.cpp

```diff
@@ -13,7 +13,11 @@
 * License for the specific language governing permissions and limitations
 * under the License.
 */
-#include "xcl2.hpp"
+
+#include "xrt/xrt_device.h"
+#include "xrt/xrt_bo.h"
+#include "xrt/xrt_kernel.h"
+
 #include <boost/filesystem.hpp>
 #include <algorithm>
 #include <vector>
```

Comment:
None of these are used in your new code.
But `<cstring>` is for std::memcmp, but you should replace that with std::equal and thus leave inclusion of `<algorithm>`

Reply:
I needed to add string.h for strcmp for the argument processing